code/_onclick/item_attack.dm

```diff
@@ -34,7 +34,7 @@
 	user.lastattacked = M
 	M.lastattacker = user
 
-	M.attacked_by(src, user)
+	. = M.attacked_by(src, user)

```

Comment:
I sure hope nothing was expecting a return value from this that it wasn't getting because oh boy that'd be fun
Why didn't this have a return value anyway augh